src/main/java/de/craften/ui/swingmaterial/MaterialFormattedTextField.java

```diff
@@ -63,8 +63,8 @@ public MaterialFormattedTextField(AbstractFormatter formatter) {
 	 * @param factory AbstractFormatterFactory used for formatting.
 	 */
 	public MaterialFormattedTextField(AbstractFormatterFactory factory) {
-		super(factory);
-		initMaterialFormattedTextField();
+		this();
+		setFormatterFactory(factory);
```

Comment:
The setFormatterFactory()>setValue()>setFormatter() method uses setText() method, that call the FloatingLabel and Line update, but setFormatterFactory() was in the constructor, causing a nullPointerException.